.pipelines/cni/cilium/cilium-cni-load-test.yaml

```diff
@@ -17,6 +17,8 @@ stages:
                 set -ex
                 make -C ./hack/swift azcfg AZCLI=az REGION=$(LOCATION)
                 make -C ./hack/swift overlay-byocni-up AZCLI=az REGION=$(LOCATION) SUB=$(SUBSCRIPTION_ID) CLUSTER=${RESOURCE_GROUP} NODE_COUNT=10 VM_SIZE=Standard_DS4_v2
+                echo "Update cluster and remove kube-proxy"
+                az aks update -n ${RESOURCE_GROUP} -g ${RESOURCE_GROUP} --kube-proxy-config ./hack/swift/kube-proxy.json

```

Comment:
I saw the note regarding the kube-proxy not being read by the make shell. What was the error message ? 
"-C" provides the directory and it should be able to read any file(given read permission)

Reply:
Testing locally, I was able to map the json config and use to bring up the cluster. I mounted the file using -v and also tried with --mount in the AZCLI docker run cmd. I also have verfied that the current directory is correct. 
However the same commands in the pipeline fails with **'kube-proxy.json is not valid file, or not accessible'**

I've been testing with different paths to the config, but azp still can't pick it up. Open to other suggestions if you have any.

Reply:
Not an expert in docker but doesn't `--rm` remove the directories ? 
When you say verified the directory, you mean you were able to see the kube-proxy file but it was not read by pipeline.

Reply:
yes, but the contents are mounted each time we run AZCLI so it should be there. I found in azp, the config file goes to **root/tmpsrc/hack/swift**, so that is path I used to list directory and to call in the create cluster command. In this run you can see that the file is there in that directory, but shows the error when I try to use it.

https://msazure.visualstudio.com/One/_build/results?buildId=73729842&view=logs&j=3713fd4c-b129-5ac5-2b38-c7386192ca24&t=bd94de76-3239-5e11-ae5b-ed0fdc2178e5&l=140

When I run this locally, the config file is directly in **root/tmpsrc** and I'm able to get it via `--kube-proxy-config root/tmpsrc/kube-proxy.json`

Reply:
Is the docker command being used ? As I see `AZCLI`  variable being passed as `az`. So if we are running the make file from the `hack/swift` dir and the file should be accessed via `./kube-proxy.json`. Did we try that already?
I think to use docker we would need to remove the `AZCLI` argument from the pipeline yaml call.

Reply:
the docker command is used, and `az` is containerized. I did try to access the file with just` ./kube-proxy.json`, but it also failed.

Reply:
Tried with cni load test pipeline: https://msazure.visualstudio.com/One/_build/results?buildId=73752234&view=logs&j=3713fd4c-b129-5ac5-2b38-c7386192ca24&t=bd94de76-3239-5e11-ae5b-ed0fdc2178e5&l=79, it seems to work. Maybe the az cli version was different for you ?

Reply:
interesting.. just tried this in the pipeline as well and it's working. Does this work for you when you run byocni-up from your dev? I get the error locally

Reply:
I tried locally but I think my preview version is outdated.(Need to check).
Getting the error invalid argument `kube-proxy-config`

Reply:
hmm, I tried updating to the latest preview and I still see the error. For now I have made separate no-kube-proxy targets that we can use specifically for the pipeline. If we need to use for a cluster in our own setup, we will just have to modify the config path to /tmpsrc/kube-proxy.json instead